components/sup/src/manager/mod.rs

```diff
@@ -270,22 +270,20 @@ impl Manager {
         opts: protocol::ctl::SvcStatus,
     ) -> NetResult<()> {
         let statuses = Self::status(&mgr.cfg)?;
-        if statuses.is_empty() {
-            req.reply_complete(net::ok());
-            return Ok(());
```

Comment:
This is what caused the zero exit status for `hab svc status <IDENT>` even if `<IDENT>` was not loaded.